tests/unit/testplan/testing/multitest/driver/myapp/test_app.py

```diff
@@ -217,6 +224,9 @@ def test_install_files(runpath):
         assert os.path.exists(app.extracts["app_path"])
         assert os.path.exists(os.path.join(app.app_path, "etc", "binary_file"))
         assert os.path.exists(os.path.join(dst, "config.yaml"))
+        assert os.path.exists(
+            os.path.join(app.app_path, "etc", "rel_path", "config.yaml")
+        )
 
```

Comment:
should also check that os.path.exists(os.path.join(app.app_path, "etc", "config.yaml"))